nixos/modules/services/x11/window-managers/dk.nix

```diff
@@ -0,0 +1,36 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.xserver.windowManager.dk;
+in
+
+{
+  options = {
+    services.xserver.windowManager.dk = {
+      enable = mkEnableOption (lib.mdDoc "dk");
+
+      package = mkOption {
```

Comment:
Use `lib.mkPackageOption`.

Reply:
Why?

Reply:
Because it is shorter and it's a typical design pattern in NixOS modules.